universum/lib/gravity.py

```diff
@@ -11,12 +11,12 @@
 
 
 class ModuleSettings:
-    def __init__(self, cls: Type['Module'], main_settings: 'HasModulesMapping') -> None:
+    def __init__(self, cls, main_settings) -> None:
```

Comment:
removed any annotations here due to lack of added value
mypy can't currently connect ModuleSettings to the underlying namespace, so type information is not preserved